brand/__init__.py

```diff
@@ -1 +1,2 @@
 from . import models
+from .tests import test_brand_mixin_model
```

Comment:
This test here looks wrong

Reply:
The test was done like this https://github.com/odoo/odoo/blob/14.0/addons/resource/__init__.py
But i will change it like @sbejaoui mentioned it and move it to account_brand